game.js

```diff
@@ -20,6 +20,7 @@ $(function () {
 
     const imgPath = 'assets/warrior/';
     const imgPathDragon = 'assets/dragon/';
+    const imgPathPotion = 'assets/';
```

Comment:
I think these paths are starting to get repetitive - we can make it more DRY by assuming that everything will be under assets.
Also, why isn't the potion icon under a separate path as warrior and dragon? We could have a "assets/potions" folder to track that.

Reply:
Thank you for this feedback. I will make those changes.

Reply:
Changes were made to the variables to be more consistent.